### PR TITLE
Create requirements.md

### PR DESCRIPTION
**Pull request by skkumaravel**
Beginning of requirements.md
1. You will later need to fork: https://github.com/CiscoDevNet/Create2018
2. Need to decide on Docker-compose or Docker Swarm (Didn't add requirements in)
    * Docker-compose is included in docker for mac and docker for windows.
3. I just made the wording up, please fix

### CODE REVIEW

**Comment by remingtonc** [comment]
If you approve of README.md then we can merge and then will duplicate to the DevNetCreate repo. Added helper scripts for using `docker stack`.

**Comment by remingtonc** [comment]
Resolves #4

File: build_images.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
```
Comment (skkumaravel) [inline build_images.sh]: This should be just one line of `docker build -t ez-dash/python ./python` no need for `cd`

File: README.md [hunk 2]
```diff
@@ -1,2 +1,49 @@
 # ez-dash
```
Comment (skkumaravel) [inline README.md]: The old title for requirements.md was the one from the DevNet workshop requirements. When you create the DevNet one, you will need to match their format. I used Hanks for a template.

Comment (remingtonc) [reply README.md]: Yup, saw that.

**Comment by skkumaravel** [comment]
Looks good!

**Pull request merged**